merlin/systems/dag/ops/operator.py

```diff
@@ -114,7 +117,11 @@ def export(
         Node_configs: list
             A list of individual configs for each step (operator) in graph.
         """
-        raise NotImplementedError
+        raise NotImplementedError(
+            "Export function only exists for the specific runtimes, that use this function"
+            f" (i.e. Triton). Please replace {self.__class__.__name__} with the appropriate "
+            f"runtime specific operator, possibly {self.__class__.__name__}Triton"
+        )
```

Comment:
This looks pretty good! Wordsmithing a little bit, how about something like:
```
Exporting an operator to run in a particular context (e.g. Triton) only makes sense when a runtime is specified. To select an operator for the appropriate runtime, replace {self.__class__.name} with a runtime-specific operator  class, possibly {self.__class__.name}Triton.
```

Reply:
I am wondering if we should even require the export function in the Base Operator. Currently only runtime specific operators would possible require an export function, thoughts?

Reply:
Yeah, I wonder about removing it too. I think I'm good with that, but I wonder if we should then move `export()` to a `RuntimeOperator` base class or something?

Reply:
So essentially we would be relocating the pipelineable inference operator to a RuntimeOperator? Instead of wanting to refactor it out, like we did in https://github.com/NVIDIA-Merlin/systems/pull/279.